src/frontend/app/shared/components/add-service-instance/specify-details-step/specify-details-step.component.ts

```diff
@@ -268,16 +268,25 @@ export class SpecifyDetailsStepComponent implements OnDestroy, AfterContentInit
             }
           });
         } else {
-          return this.createServiceInstance(p, this.modeService.isEditServiceInstanceMode());
+          return this.createServiceInstance(p);
         }
       }),
       filter(s => !s.creating && !s.fetching),
       combineLatest(this.store.select(selectCreateServiceInstance)),
       first(),
       switchMap(([request, state]) => {
-        if (request.error) {
+        if (this.modeService.isEditServiceInstanceMode()) {
+          const updatingInfo = request.updating[UpdateServiceInstance.updateServiceInstance];
+          if (!!updatingInfo && updatingInfo.error) {
+            return observableOf({
+              success: false,
+              message: `Failed to update service instance: ${updatingInfo.message}`
```

Comment:
I get the message: `The service broker rejected the request to https://app-autoscaler-broker.cf-dev.io/v2/service_instances/5e44b48e-c25e-4c76-963a-ed419533a033. Status Code: 404 Not Found, Body: 404 page not found` this isn't very friendly and isn't particularly useful. Maybe we should just show the `Failed to update service instance`?

Reply:
I returned the original exception because the exception could basically explain why the request failed. In this case, the test broker doesn't implement the endpoint to update params.